shaders/src/getters.fs

```diff
@@ -54,6 +54,23 @@ vec3 getWorldViewVector() {
     return shading_view;
 }
 
+bool isPerspectiveProjection() {
+    return frameUniforms.clipFromViewMatrix[2].w != 0.0;
+}
+
+/** @public-api */
+vec3 getWorldViewVectorWithMagnitude() {
```

Comment:
What about we call this simply `getWorldViewVector()` and we document that it's not normalized.

Reply:
I ended up reverting the changes to the public API. With this I reverted our ortho fixes for fog and punctual light too